api/HTMLElement.json

```diff
@@ -2458,7 +2458,9 @@
             ],
             "support": {
               "chrome": {
-                "version_added": "133"
+                "version_added": "133",
+                "partial_implementation": true,
+                "notes": "This doesn't apply the focus ordering change. See [bug 383343310](https://crbug.com/383343310)."
```

Comment:
Can you elaborate what "focus ordering change" refers to here, and why it is specific to the source option?

PS: While we're here, maybe we could add this `spec_url`: https://html.spec.whatwg.org/multipage/dom.html#dom-showpopoveroptions-source

Reply:
If I understand this correctly, we could rephrase it for users as follows:

> When using this option, the focus order doesn't change, so the popover does not become the next focus element.

You might be able to help out making this more idiomatic.

Reply:
```suggestion
                "notes": "When using this option, the focus order doesn't change, so the popover does not become the next focus element. See [bug 383343310](https://crbug.com/383343310)."
```

Reply:
That sounds good and yes I'll add the spec URL too

Reply:
Sorry for delay I've done that now.